src/App/AppModLeds.h

```diff
@@ -234,18 +233,17 @@ class AppModLeds:public Module {
     // FastLED.addLeds<NEOPIXEL, 6>(leds, 1); 
     FastLED.addLeds<NEOPIXEL, DATA_PIN>(ledsP, NUM_LEDS_FastLed); 
 
-    #ifdef USERMOD_E131
-      e131mod->addWatch(1, "bri", 256);
-      e131mod->addWatch(2, "fx", effects.size());
-    #endif
+    mdl->addWatch(1, "bri", 256);
+    mdl->addWatch(2, "fx", effects.size());
+    mdl->addWatch(3, "projection", Projections::count);
```

Comment:
If you are going to add a new "common" param then the effects will need to be updated to use new channel. Might be best that the per effect parameters start at 10